Add static and dynamic tables capability to plugin (#4)
turbot/steampipe-plugin-salesforce: docs/tables/{object_name}.md

+--------------------+--------------------+------------+------+---------------------------+--------------------+---------------------------+---------------------+---------------------------+
| a005j000006vluAAAQ | 0055j000003GU6IAAW | false      | AAP1 | 2022-01-13T14:18:28+05:30 | 0055j000003GU6IAAW | 2022-01-13T14:18:28+05:30 | 0055j000003GU6IAAW  | 2022-01-13T14:18:28+05:30 |
| a005j000006vluBAAQ | 0055j000003GU6IAAW | false      | APP2 | 2022-01-13T14:19:17+05:30 | 0055j000003GU6IAAW | 2022-01-13T14:19:17+05:30 | 0055j000003GU6IAAW  | 2022-01-13T14:19:17+05:30 |
| a005j000006vluFAAQ | 0055j000003GU6IAAW | false      | APP1 | 2022-01-13T14:18:58+05:30 | 0055j000003GU6IAAW | 2022-01-13T14:18:58+05:30 | 0055j000003GU6IAAW  | 2022-01-13T14:18:58+05:30 |
+--------------------+--------------------+------------+------+---------------------------+--------------------+---------------------------+---------------------+---------------------------+
```

However, please note that this could be slow depending on how many objects are in your environment.

> **NOTE:** Salesforce custom object and custom columns are suffixed with `__c`

Reviewer: ```suggestion
**NOTE:** Salesforce custom objects and fields are always suffixed with `__c`
```

+---------------------+--------------------------+--------------------------+
| created_by_id       | text                     | The Created By ID.       |
| created_date        | timestamp with time zone | The Created Date.        |
| id                  | text                     | The Record ID.           |
| is_deleted          | boolean                  | The Deleted.             |
| last_modified_by_id | text                     | The Last Modified By ID. |
| last_modified_date  | timestamp with time zone | The Last Modified Date.  |
| name                | text                     | The Custom APP Name.     |
| owner_id            | text                     | The Owner ID.            |
| system_modstamp     | timestamp with time zone | The System Modstamp.     |
+---------------------+--------------------------+--------------------------+
```

### Get all the values from salesforce_custom_app\_\_c

```sql
select
  *
from
  salesforce_custom_app__c;
```

### List custom apps added in last 24 hrs

```sql
select
  id,
  name,
  owner_id
from
  salesforce_custom_app__c
where
  created_date = now() - interval '24 hrs';
```

> ### Get details for a custom app with specific id

Reviewer: ```suggestion
### Get details for a custom app by ID
```
